x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/anomaly_detection_panel.tsx

```diff
@@ -0,0 +1,209 @@
+      const stats = getStatsBarData(jobsSummaryList);
+      setIsLoading(false);
+      setErrorMessage(undefined);
+      setStatsBarData(stats);
+      setGroupsCount(Object.values(jobsGroups).length);
+      setGroups(jobsGroups);
+      setJobsList(jobsWithTimerange);
+      loadMaxAnomalyScores(jobsGroups);
+    } catch (e) {
+      setErrorMessage(JSON.stringify(e));
+      setIsLoading(false);
+    }
+  };
+
+  const loadMaxAnomalyScores = async (groupsObject: Groups) => {
+    const groupsList: Group[] = Object.values(groupsObject);
+    const scores = getDefaultAnomalyScores(groupsList);
+
+    try {
+      const promises: any[] = [];
+      groupsList.forEach((group, i) => {
+        if (group.jobIds.length > 0) {
+          scores[group.id].index = i;
+          promises.push(
+            mlResultsService.getOverallBucketScores(
+              group.jobIds,
+              1,
+              group.earliest_timestamp,
+              group.latest_timestamp
+            )
+          );
+        }
+      });
+
+      const results = await Promise.all(promises);
+      const tempGroups = Object.assign({}, { ...groupsObject });
+      // Check results for each group's promise index and update state
+      Object.keys(scores).forEach(groupId => {
+        const resultsIndex = scores[groupId] && scores[groupId].index;
+        const promiseResult: { success: boolean; results: { [key: string]: number } } =
+          resultsIndex !== undefined && results[resultsIndex];
+        if (promiseResult.success === true && promiseResult.results !== undefined) {
+          const maxScore = Math.max(...Object.values(promiseResult.results));
+          scores[groupId] = { maxScore };
+          tempGroups[groupId].max_anomaly_score = maxScore;
+        }
+      });
+
+      setGroups(tempGroups);
+    } catch (e) {
+      toastNotifications.addDanger(
+        i18n.translate(
+          'xpack.ml.overview.anomalyDetection.errorWithFetchingAnomalyScoreNotificationErrorMessage',
+          {
+            defaultMessage: 'An error occurred fetching anomaly scores: {error}',
+            values: { error: JSON.stringify(e) },
+          }
+        )
+      );
+    }
+  };
+
+  useEffect(() => {
+    loadJobs();
+  }, []);
+
+  const onRefresh = () => {
+    loadJobs();
+  };
+
+  const errorDisplay = (
+    <Fragment>
+      <EuiCallOut
+        title={i18n.translate('xpack.ml.overview.anomalyDetection.errorPromptTitle', {
+          defaultMessage: 'An error occurred getting the anomaly detection jobs list.',
+        })}
+        color="danger"
+        iconType="alert"
+      >
+        <pre>{JSON.stringify(errorMessage)}</pre>
+      </EuiCallOut>
+    </Fragment>
+  );
+
+  return (
+    <EuiPanel className="mlOverviewPanel">
+      {typeof errorMessage !== 'undefined' && errorDisplay}
+      {isLoading && <EuiLoadingSpinner />}   
+      {isLoading === false && typeof errorMessage === 'undefined' && groupsCount === 0 && (
```

Comment:
As previously reported, when there are no Anomaly Detection jobs, rather than the expected `EuiEmptyPrompt` I am seeing:

![image](https://user-images.githubusercontent.com/7405507/65155016-67ffb580-da24-11e9-9cd5-a22655b25a16.png)

Reply:
Fixed with 59c980893ca72c667bda1921584b74d5907f35bf
(ensures removal of `ungrouped` when there are no ungrouped jobs)